### PR TITLE
Create python-package.yml

### LINKED ISSUE #15: Set up CI using Github Actions
We are no longer using Travis CI and need to instead set up the CI pipeline using Github Actions.

### PR DESCRIPTION
**Pull request by tamarl08**
In this PR I used github template instead of creating a yml file. please review this before [the previous PR](https://github.com/EducationalTestingService/match/pull/17) and if this is fine I will close it.
The tests include python 3.7,3.8,3.9 as opposed to the original testing with only 3.7. The tests fail on python 3.10.

### CODE REVIEW

**Review by slava92 — DISMISSED** [review]
Looks great!

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```
Comment (desilinguist) [inline .github/workflows/python-package.yml]: We should rename this right since this action is not about packaging?

**Review by desilinguist — CHANGES_REQUESTED** [review]

**Comment by desilinguist** [comment]
This looks great, @tamarl08 ! I just had a minor comment. Also, I think we need to resolve at least the branch naming discussion on the discussions [page](https://github.com/EducationalTestingService/match/discussions) before we merge this PR.

**Review by desilinguist — DISMISSED** [review]
LGTM!

**Comment by desilinguist** [comment]
Can we get rid of the `.travis.yml` file too please?

**Review by desilinguist — DISMISSED** [review]
Looks great!

**Review by desilinguist — APPROVED** [review]

**Pull request merged**